CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and includes an additional section for migration notes.
 ## [Unreleased]
 
 ### Changed
```

Comment:
You currently have 2 ### Changed sections. I would condense to the one that already exists below the ###Added section.

Reply:
Fixed